woocommerce/woocommerce-android
# [UOE] Track edit button tap

Part of: #6711

### Description
This PR tracks edit button tap event:

- order_edit_button_tapped 
   with params:
   - has_multiple_shipping_lines: Bool
   - has_multiple_fee_lines: Bool

### Testing instructions
TC1
- Run OrderDetailViewModelTest

TC2
- Go to the Orders
- Select an order with multiple shipping lines and without fee lines
- Press Edit
- Check  `🔵 Tracked: order_edit_button_tapped, Properties: { "has_multiple_fee_lines": false, "has_multiple_shipping_lines": true` is triggered

TC3
- Go to the Orders
- Select an order without shipping lines and multiple fee lines
- Press Edit
- Check  `🔵 Tracked: order_edit_button_tapped, Properties: { "has_multiple_fee_lines": true, "has_multiple_shipping_lines": false` is triggered

TC4
- Go to the Orders
- Select an order with multiple shipping lines and multiple fee lines
- Press Edit
- Check  `🔵 Tracked: order_edit_button_tapped, Properties: { "has_multiple_fee_lines": true, "has_multiple_shipping_lines": true` is triggered


TC5
- Go to the Orders
- Select an order without shipping lines and without fee lines
- Press Edit
- Check  `🔵 Tracked: order_edit_button_tapped, Properties: { "has_multiple_fee_lines": false, "has_multiple_shipping_lines": false` is triggered
